packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1830,33 +1830,8 @@ abstract class SingleChildRenderObjectWidget extends RenderObjectWidget {
   SingleChildRenderObjectElement createElement() => SingleChildRenderObjectElement(this);
 }
 
-/// A superclass for [RenderObjectWidget]s that configure [RenderObject] subclasses
-/// that have a single list of children. (This superclass only provides the
-/// storage for that child list, it doesn't actually provide the updating
-/// logic.)
-///
-/// Subclasses must return a [RenderObject] that mixes in
-/// [ContainerRenderObjectMixin], which provides the necessary functionality to
-/// visit the children of the container render object (the render object
-/// belonging to the [children] widgets). Typically, subclasses will return a
-/// [RenderBox] that mixes in both [ContainerRenderObjectMixin] and
-/// [RenderBoxContainerDefaultsMixin].
-///
-/// See also:
-///
-///  * [Stack], which uses [MultiChildRenderObjectWidget].
-///  * [RenderStack], for an example implementation of the associated render
-///    object.
-///  * [SlottedMultiChildRenderObjectWidgetMixin], which configures a
-///    [RenderObject] that instead of having a single list of children organizes
-///    its children in named slots.
-abstract class MultiChildRenderObjectWidget extends RenderObjectWidget {
-  /// Initializes fields for subclasses.
-  ///
-  /// The [children] argument must not be null and must not contain any null
-  /// objects.
-  const MultiChildRenderObjectWidget({ super.key, this.children = const <Widget>[] });
-
+/// Mixin to allow sharing the required properties of [MultiChildRenderObjectWidget]
+mixin MultiChildRenderObjectWidgetMixin {
```

Comment:
Is this change needed?

Reply:
Not necessarily, but again I wanted to share these properties with classes that no longer extend this

Reply:
If the goal is to get the children getter, I think put it in AbstractFlex and use 
{@template ..} and {@endtemplate} to reuse documentation would be fine